schema.json

```diff
@@ -0,0 +1,496 @@
+            "type": "array",
+            "minItems": 2,
+            "maxItems": 2,
+            "uniqueItems": true,
+            "items": {
+                "type": "object",
+                "examples": [
+                    {
+                        "src": "https://bitrise-steplib-collection.s3.amazonaws.com/step-archives/",
+                        "type": "zip"
+                    },
+                    {
+                        "src": "source/git",
+                        "type": "git"
+                    }
+                ],
+                "required": [
+                    "type",
+                    "src"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                    "src": {
+                        "type": "string",
+                        "pattern": "(^https://bitrise-steplib-collection.s3.amazonaws.com/step-archives/$)|(^source/git$)"
+                    },
+                    "type": {
+                        "type": "string",
+                        "pattern": "(^git$)|(^zip$)"
+                    }
+                }
+            }
+        },
+        "assets_download_base_uri": {
+            "type": "string",
+            "pattern": "^https://bitrise-steplib-collection.s3.amazonaws.com/steps$",
+            "examples": [
+                "https://bitrise-steplib-collection.s3.amazonaws.com/steps"
+            ]
+        },
+        "steps": {
+            "type": "object",
+            "propertyNames": {
+                "minLength": 1
+            },
+            "additionalProperties": {
+                "type": "object",
+                "required": [
+                    "info",
+                    "latest_version_number",
+                    "versions"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                    "info": {
+                        "type": "object",
+                        "additionalProperties": false,
+                        "properties": {
+                            "removal_date": {
+                                "type": "string"
+                            },
+                            "deprecate_notes": {
+                                "type": "string"
+                            },
+                            "asset_urls": {
+                                "type": "object",
+                                "additionalProperties": false,
+                                "maxProperties": 2,
+                                "properties": {
+                                    "icon.svg": {
+                                        "type": "string"
+                                    },
+                                    "icon.png": {
+                                        "type": "string"
+                                    }
+                                }
+                            },
+                            "maintainer": {
```

Comment:
we have a fix set of maintainers currently, pls validate them

Reply:
there is no maintainer field in the spec yet, what is the current list?